FAQ.md

```diff
@@ -6,27 +6,37 @@ _The first rule of FAQ is don't bikeshed the FAQ, leave that for
 Please contribute to the FAQ! Found an explanation in an issue or pull request helpful?
 Summarize the question and quote the reply, linking back to the original comment.
 
+## Concepts
 * [Does `dep` replace `go get`?](#does-dep-replace-go-get)
 * [Why is it `dep ensure` instead of `dep install`?](#why-is-it-dep-ensure-instead-of-dep-install)
+* [What is a direct or transitive dependency?](#what-is-a-direct-or-transitive-dependency)
+
+## Configuration
 * [What is the difference between Gopkg.toml (the "manifest") and Gopkg.lock (the "lock")?](#what-is-the-difference-between-gopkgtoml-the-manifest-and-gopkglock-the-lock)
```

Comment:
> I would argue that Gopkg.lock is not a configuration file and the user should not update it manually (except in very limited cases).

I think this is getting into the "bikeshedding the FAQ" territory. 😀 How about we move the question into a spot for now, and then circle to split this into the question my heart desired "What are dep's files and how should I use them?" (answer: edit the manifest and leave the lock alone) and the answer we have to the unspoken question "Why didn't you just shove everything into a single file?".